bundles/org.pitest.pitclipse.launch.ui/src/org/pitest/pitclipse/launch/ui/PitMigrationDelegate.java

```diff
@@ -89,6 +83,19 @@ private static IResource getResource(ILaunchConfiguration config)
         return resource;
     }
 
+    private static IJavaElement getJavaProject(String typeName, String container, IJavaElement element,
```

Comment:
That's not a big deal but I'd prefer the `element` parameter to become a variable local to the `getJavaProject` method. With the current program the argument will always be null so I think that keeping the parameter is only confusing for the reader.

Reply:
You're perfectly right!